### PR TITLE
fix(ui): update and fix rollup

### PR DESCRIPTION
**Pull request by geigerzaehler**
While trying to integrate the `@walletconnect/client` I ran into some issues with rollup that this PR adresses

* We change the package name of some rollup plugins to get the latest versions
* We switch the order of the `commonjs` and `typescript` plugins. Now the `browser` field in NPM packages is respected and we’re using the browser builds for these packages where node builds may fail.

### CODE REVIEW

**Review by xla — CHANGES_REQUESTED** [review]
The documentation of the node-resolve plugin indicates that there needs to be a certain order when used with **commonjs**: https://github.com/rollup/plugins/tree/master/packages/node-resolve#using-with-rollupplugin-commonjs

**Comment by geigerzaehler** [comment]
> The documentation of the node-resolve plugin indicates that there needs to be a certain order when used with **commonjs**: https://github.com/rollup/plugins/tree/master/packages/node-resolve#using-with-rollupplugin-commonjs

Yes. It seem that it was only necessary to flip the order of `commonjs` and `typescript`. Now `commonjs` still comes after `nodeResolve`.

**Review by xla — APPROVED** [review]
🐟 🈹 ⚙ 🔝

File: rollup.app.js [hunk 1]
```diff
@@ -28,20 +28,16 @@ export default {
...
-    // https://github.com/rollup/rollup-plugin-commonjs
     resolve({
       browser: true,
       dedupe: importee =>
         importee === "svelte" || importee.startsWith("svelte/"),
     }),
+
     commonjs(),
 
+    typescript(),
```
Comment (rudolfs) [inline rollup.app.js:39]: Should we maybe document here that the order is important?

**Review by rudolfs — APPROVED** [review]

**Pull request merged**